webpack.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    mode: 'development',
+    devtool: 'eval-source-map',
+    resolve: {
+      extensions: ['.ts', '.js'],
+    },
+    module: {
+      rules: [
+        {
+
+          test: /\.ts$/,
+          exclude: [/node_modules/],
+          use: [
+            {
+              loader: 'ts-loader',
+              options: {
+                transpileOnly: true,
+              },
+            },
+          ],
+        },
+      ],
+    },
+  }
```

Comment:
Fix the indentation of the brackets, they should go in the same column of the opening bracket

Reply:
nevermind... the identation is ok